app/assets/stylesheets/application.scss

```diff
@@ -552,3 +552,9 @@ html.turbolinks-progress-bar::before {
   display: block;
   float: left;
 }
+
+.ui-autocomplete {
+  height: 200px;
```

Comment:
px is fine here, maybe increase it to 300px though so it won't clip so soon.
However, you should use `max-height` instead of `height` so there won't extra white space when there are few options (for example, in the architecture page, or after the first auto complete (i.e. for the operators)).